templates/leaderboard/includes/list.html

```diff
@@ -9,7 +9,7 @@
                     <img class="profile-crop" src="{{ leader_context.avatar_url }}" alt="{{ leader_context.name }}" />
                     <br/>
                     <h4 class="chart-name">#{{ forloop.counter }}: {{ leader_context.name }}</h4>
-                    <h4 class="chart-amount">${{ leader_context.amount|notrail_floatformat:2|intcomma }}</h4>
+                    <h4 class="chart-amount">{{ leader_context.percentage|notrail_floatformat:2|intcomma }}%</h4>
```

Comment:
Currently users are being sorted in this leaderboard by the amount that they've earned on PerDiem. By removing the amount, it will make things less clear to a user viewing the page what the meaning of the ordering is.

Should we change the ordering so that higher percentages come earlier in the list? If we do that, it would have a weird effect of shooting new investors to the top of the leaderboard that happened to get lucky rather than long-term investors that have used the platform a lot.

What do you think?

Reply:
Ahh. Yeah, that makes sense. Most earned would be better than %.

Reply:
For now I'm just going to have it present the top investors without an amount or percentage. Still trying to work out the public investment part. Want to try and have the earnings be internal and see how people react to it